test/Sema/conformance_availability_warn.swift

```diff
@@ -52,3 +52,33 @@ func passAvailableConformance1a(x: HasAvailableConformance1) {
   _ = x.isGalloping
   _ = UsesHorse<HasAvailableConformance1>.self
 }
+
+// Explicit unavailability
+public struct HasAvailableConformance2 {}
```

Comment:
Minor: I would find this a bit clearer if it were named `HasUnavailableConformance` (it looks like the equivalent struct is named that way in `conformance_availability.swift`).